shared/datatypes.yaml

```diff
@@ -0,0 +1,29 @@
+---
+# This file makes the correspondance between extensions on Zenodo (first element) and Galaxy datatypes (second element)
+# Keep the alphabetical order
+bam: bam
+bed: bed
+bedGraph: bedgraph
```

Comment:
bg

Reply:
So you have your own format :smile:

Reply:
Joking aside, is it bedgraph?

Reply:
Yes its a pretty common ext for bedgraph.